test/test_caffe2.cc

```diff
@@ -638,7 +638,7 @@ def fun(float(B, N, M) X, float(B, M, K) Y) -> (Z)
   Workspace w_test;
   init_ws(w_test);
   Argument tcArg = MakeArgument<string>("def", tc);
```

Comment:
but here the `def` argument is written

Reply:
yes, thanks for catching, I am still making changes